XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
You tell me! :) As I see it, we have to options here:

- full parent folders tree search
- search up to solution folder

Personally, I prefer scenario 1, but I'm ok changing to scenario 2 if that's the general consensus!